site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -37,120 +34,94 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
+2. The "triggered by hover" definition: any **pop-up window/dialog**, containing **any type of content**, that **opens when a user mouse-hovers** an element.
+3. The "semantic" definition: A popover containing **non-essential**, **auxiliary** information about a context element.
 
-The existing popover API includes the concept of "invoking elements":
+Definition <span>#</span>1 is too limiting, and does not capture common use cases that include non-textual content. A good example is the window that appears when you hover over a username on Github:
 
-```html
-<button popovertarget=foo popovertargetaction=show>
-  Click me to show a popover
-</button>
-<div popover id=foo> I'm a popover </div>
-```
+  ![Github user information tooltip](/images/github-tooltip.png)
 
-The `popovertarget` attribute can be used on buttons (and `<input type=button>`), and it points to the idref of an element with the `popover` attribute. In this configuration, clicking the button automatically shows the popover, without requiring Javascript. In addition, this linkage ensures that the accessibility tree is properly updated to represent the connection. The `popovertargetaction` attribute determines what action to take when the button is activated: `show`, `hide`, or `toggle`.
+Definition <span>#</span>2 is mis-guided. Many things can be "hover-triggered" without being tooltips. A great example is a hover-triggered menu, where hovering over a sub-menu item brings up the sub-menu. Most people agree that the sub-menu is **not** a tooltip. It just happens to be hover-triggered.
 
-## Add the `interest` value
+That leaves definition <span>#</span>3, which I believe is the correct definition. A tooltip should not contain directly-necessary information - that should be placed into the page content directly. It should provide auxiliary or extra information that is available only when the user wants to dig deeper into some content. And that extra information should be able to contain more than just text, since the auxiliary information might very well contain or even **be** an image or other non-text content.
 
-To support hover triggering, a new value is added: `popovertargetaction=interest`. The name of "interest" was [chosen](https://github.com/openui/open-ui/issues/767#issuecomment-1654177227) to represent the fact that the popover will be triggered not just via mouse-hover, but also by keyboard focus or touchscreen long-press. When set to the `interest` value, the popover will be **shown** when the invoking element is **a)** hovered with the mouse, **b)** focused, or **c)** long-pressed (mobile).
+Note, importantly, that a tooltip **does not need** to be triggered by hover to be a tooltip. Also note that all things that are hover-triggered are not tooltips. A tooltip is defined more by **its content** than how it gets **activated**.
```

Comment:
remove `more` as it's not adding anything

Reply:
Removed `more` but had to re-jigger the ending to be parallel.